emcc.py

```diff
@@ -822,7 +822,8 @@ def run(args):
       print(shared.shlex_join(parts[1:]))
     return 0
 
-  shared.check_sanity(force=DEBUG)
+  if os.environ.get('EMCC_SKIP_SANITY_CHECK') != '1':
```

Comment:
this is already checked in the called function, except when `force` is set. Does this change then make `EMCC_SKIP_SANITY_CHECK` override `EMCC_DEBUG`? seems ok to me, if there is a reason, but is it related to this PR?

Reply:
Yes this is a little annoying. 

We want `EMCC_SKIP_SANITY_CHECK` to take precedence over "foce_because_emcc_debug" but not over "force_because_of_command_line".    I guess they are two difference kinds of forcing and this seemed like smallest change to make that happen.  I will see if I can refactor again to make it more logical.